### PR TITLE
Produce basic instructions on Backup and Recovery

### PR DESCRIPTION
**Pull request by MaxHillebrand**
Contains:
* explanation why backup is important
* explanation and tutorial on password protection
* explanation and tutorial on the mnemonic seed
* explanation and tutorial on the local backup
* tutorial on import backup  for all OS

Needs in addition / questions:
* tutorial on how to encrypt the backup
* screen shots of wallet interface, seed and backup files

[Comments by Manfred](https://github.com/bisq-network/bisq-docs/commit/2b1b074245f077b4ae1e03c05d3fc0bb84fbd19d#commitcomment-31622517)

### CODE REVIEW

**Comment by m52go** [comment]
This doc should be close to complete now.

I tested most of the instructions in the doc, so the instructions should be fairly accurate. I'm not 100% sure about the wallet section, however, so please pay special attention to that when reviewing.

File: backup-recovery.adoc [hunk 1]
```diff
@@ -1,26 +1,101 @@
...
+
+**Account details** consist of all the payment methods you've set up and their associated account numbers, bank information, addresses, etc. You can export this data from within Bisq on the `National currency accounts` and `Altcoin accounts` tabs of the `Account` screen. Click `Export accounts` and Bisq will output a file that you can use to import these details later.
+
+**Account aging** establishes the age of your payment accounts. To reduce fraud, newer payment accounts have lower trading limits.
+
+To include account aging with your account details import, replace the `btc_mainnet/keys/sig.key` file with the `sig.key` file from your backup. Be sure you don't have any open offers, trades, or disputes before changing `sig.key`, otherwise you won't be able to respond to those activities.
+
+If you don't have an accounts export file, but you have a full backup of the data directory, you can still fully restore your accounts details and accounts aging by manually copying these files to your data directory from a backup:
+
+* `btc_mainnet/db/AccountAgeWitnessStore` (salt)
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: AccountAgeWitnessStore is not needed. That contains the public hashes but not the salt which is part of the payment account data.

File: backup-recovery.adoc [hunk 2]
```diff
@@ -1,26 +1,101 @@
...
+* `btc_mainnet/keys/sig.key` (key used to sign p2p messages)
+* `btc_mainnet/keys/UserPayload` (account details)
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: btc_mainnet/keys/UserPayload -> btc_mainnet/db/UserPayload (account details including salt)

File: backup-recovery.adoc [hunk 3]
```diff
@@ -1,26 +1,101 @@
...
+
+=== Details
+
+In your Bisq data directory, you will see a few items. The most notable file is `bisq.log`, which is the Bisq program log file. Checking it can be helpful when you have issues with the Bisq program.
+
+The other notable item is `btc_mainnet`, which is the main folder containing data for your payment accounts, Tor setup, and wallet.
+
+== Payment Accounts
+
+There are 2 components to payment accounts on Bisq: account details and account aging.
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: Account age witness is only relevant for fiat accounts. For altcoin accounts it does not exits as there is no chargeback risk.

Comment (ManfredKarrer) [reply backup-recovery.adoc]: Maybe add a link to the account age witness doc/spec for further details?

File: backup-recovery.adoc [hunk 4]
```diff
@@ -1,26 +1,101 @@
...
+
+IMPORTANT: Before manually making any changes to Bisq data files, please remember to back up the current state of your Bisq data directory and make sure the Bisq program is closed.
+
+== Wallet Data
+
+Your wallet data is stored in `btc_mainnet/wallet`.
+
+Wallet issues (e.g., incorrect balance showing, corrupted/out-of-sync SPV file) can often be solved by resyncing the SPV chain and restarting the Bisq program. You can do this by clicking the `Delete SPV file and resync` button on the `Network info` tab of the `Settings` screen. If you can't access this screen, you can just delete the `bisq.spvchain` file in `btcmainnet/wallet` (just be advised that you will need to manually restart Bisq twice to complete the resync).
+
+If issues persist, you can try to restore a backup wallet file from `btc_mainnet/wallet/backup/backups_bisq_BTC_wallet`. This folder contains 20 of your most recent wallet backups (backups are made on a rolling basis). Replace the `bisq_BTC.wallet` wallet file in `btc_mainnet/wallet` with a backup wallet file and rename it to `bisq_BTC.wallet`. Start Bisq and see if your issue is fixed. If not, repeat with another backup.
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: If using a backup file you need to delete the spv file as well.

File: backup-recovery.adoc [hunk 5]
```diff
@@ -1,26 +1,101 @@
...
+
+**Make sure you don't have any open offers, trades, or disputes before changing your onion address, as doing so will make you unreachable.**
+
+IMPORTANT: Before manually making any changes to Bisq data files, please remember to back up the current state of your Bisq data directory and make sure the Bisq program is closed.
+
+== Wallet Data
+
+Your wallet data is stored in `btc_mainnet/wallet`.
+
+Wallet issues (e.g., incorrect balance showing, corrupted/out-of-sync SPV file) can often be solved by resyncing the SPV chain and restarting the Bisq program. You can do this by clicking the `Delete SPV file and resync` button on the `Network info` tab of the `Settings` screen. If you can't access this screen, you can just delete the `bisq.spvchain` file in `btcmainnet/wallet` (just be advised that you will need to manually restart Bisq twice to complete the resync).
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: Re-sync can take a while and CPU will be at 100% for that time. The more transactions you had and the older the wallet is the longer it takes. Bisq UI is not responsive at that time as the wallet resync takes up all CPU resources.

File: backup-recovery.adoc [hunk 6]
```diff
@@ -1,26 +1,101 @@
...
+
+If issues persist, you can try to restore a backup wallet file from `btc_mainnet/wallet/backup/backups_bisq_BTC_wallet`. This folder contains 20 of your most recent wallet backups (backups are made on a rolling basis). Replace the `bisq_BTC.wallet` wallet file in `btc_mainnet/wallet` with a backup wallet file and rename it to `bisq_BTC.wallet`. Start Bisq and see if your issue is fixed. If not, repeat with another backup.
+
+In extreme cases, if neither resyncing the SPV file nor restoring a wallet backup solves the problem, you might need to set up a new data directory. Make sure you don't have any open offers, trades, or disputes before doing this, as you won't be able to respond to these activities any longer.
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: Hm... I think if the wallet is completely fucked up then restore from seed words is the emergency tool to get the funds restored. Switching to to a new data dir is more if there are stange bugs. There is a wallet db file which is important for the internal wallet management (db/AddressEntryList). If that is out of sync best is to start over with a new data dir.

File: backup-recovery.adoc [hunk 7]
```diff
@@ -1,26 +1,101 @@
...
+
+Here's how to do it:
+
+* make sure you have your Bisq wallet's <<secure-wallet#, seed words>>
+* close Bisq
+* delete the whole `Bisq` directory (after backing it up, just in case)
+* start Bisq (it will generate a fresh new data directory)
+* restore your wallet from seed words in the Bisq interface (on the `Wallet seed` tab of the `Account` screen)
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: I recommend to start with a new wallet. Old wallets get heavy as they contian many txs and spv sync takes longer as well normal startup becomes heavier over time. Easiest is that the user first send out the funds to an external wallet and then send the funds to the new wallet.

File: backup-recovery.adoc [hunk 8]
```diff
@@ -1,26 +1,101 @@
...
+* optional: restore payment accounts and/or Tor keys through the processes outlined above
+
+**Make sure you don't have any open offers, trades, or disputes before doing this.**
+
+Please do not try to copy an old wallet backup folder (i.e., `btc_mainnet/wallet`) into a new data directory, as that will cause internal database files to be out of synx—just restore from seed words.
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: The backup folders would not cause any issue. I think u can delete that sentence.

File: backup-recovery.adoc [hunk 9]
```diff
@@ -1,26 +1,101 @@
...
+
+== Tor State
+
+Bisq runs as a Tor hidden service to protect your privacy.
+
+As you do trades, you'll build _local reputation_ whereby Bisq will show you how many trades you've completed with other peers. The idea is that if one trade with someone went well, it's reasonable to expect your next trade with that person will go well too, creating a rudimentary sort of reputation while maintaining anonymity. This mechanism is tied to your onion address.
+
+If you want to migrate a particular onion address, you can replace the `btc_mainnet/tor/hiddenservice` folder in your data directory with the one from your backup.
+
+If you want to change your onion address, just delete the `btc_mainnet/tor/hiddenservice`. The next time you start Bisq, you'll have a different onion address.
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: Maybe add a note that you gain more privacy with that as past trades are not connected with new trades or offers via the onion address (only visible to past traders) but u will lose the reputation.

File: backup-recovery.adoc [hunk 10]
```diff
@@ -1,26 +1,101 @@
...
+
+The other notable item is `btc_mainnet`, which is the main folder containing data for your payment accounts, Tor setup, and wallet.
+
+== Payment Accounts
+
+There are 2 components to payment accounts on Bisq: account details and account aging.
+
+**Account details** consist of all the payment methods you've set up and their associated account numbers, bank information, addresses, etc. You can export this data from within Bisq on the `National currency accounts` and `Altcoin accounts` tabs of the `Account` screen. Click `Export accounts` and Bisq will output a file that you can use to import these details later.
+
+**Account aging** establishes the age of your payment accounts. To reduce fraud, newer payment accounts have lower trading limits.
```
Comment (ManfredKarrer) [inline backup-recovery.adoc]: To reduce fraud, -> To reduce fraud risk,

Comment (ManfredKarrer) [reply backup-recovery.adoc]: maybe link at To reduce fraud risk to the doc

**Pull request merged**